django_mongodb/client.py

```diff
@@ -0,0 +1,8 @@
+from django.db.backends.base.client import BaseDatabaseClient
+
+
+class DatabaseClient(BaseDatabaseClient):
+    executable_name = "django_mongodb"
```

Comment:
this class implements the django sqlshell command, which I personally don't use, so this is optional, but should be fairly easy